src/components/Form/Form.tsx

```diff
@@ -34,10 +36,24 @@ export const Form: FC<FormProps> = ({ activity }) => {
   const handleEvaluateFormRules = async (
     response: Array<AnswerInput>
   ): Promise<Array<QuestionRuleResult>> => {
+    addSentryBreadcrumb({
+      category: BreadcrumbCategory.EVALUATE_FORM_RULES,
+      data: {
+        response,
+        form,
```

Comment:
We need to be a bit careful about what information we forward to Sentry + see we don't send an overload of information so I'd only send `response` and `form_id`. That's definitely sufficient for us.